crates/daphne/src/taskprov.rs

```diff
@@ -572,4 +572,57 @@ mod test {
     }
 
     test_versions! { resolve_advertised_task_config_expect_abort_unrecognized_vdaf }
+
```

Comment:
We already had this test: https://github.com/cloudflare/daphne/pull/596/files#diff-e8ba4a25150d4a6f4c52e3735a7b4b04a034a096b35163063bcbe9d4914b8dadR512

It looks like we deleted it in https://github.com/cloudflare/daphne/pull/600. Let's double check that we didn't inadvertenly remove other code in that PR.

"we_accept_all_dp_configuration_parameters" is an odd name for a test. (Who is "we"?)

I think it would be better to test resolve_advertised_task_config().